tests/applications/flag_control_mixin_test.py

```diff
@@ -16,16 +28,18 @@ def test_start_twice():
     app = DummyApp()
```

Comment:
We have adopted the unofficial convention that when we make tests that are supposed to verify an exception is raised, we reflect it in the test name. So in this case, this test could be called something like `test_start_twice_raises()`